Cue/app/common/DeckThumbnail.js

```diff
@@ -57,6 +86,22 @@ export default class DeckThumbnail extends React.Component {
       // Display "Updating..."?
     }
 
+    let inset
+    if (!this.props.hideInsets) {
+      if (this.props.deck.accession === 'private') {
+        if (this.props.deck.public) {

```

Comment:
Shouldn't we show the public inset regardless of whether or not the user owns the deck?